pipelines/build_pipeline_pass.yaml

```diff
@@ -11,6 +11,15 @@ spec:
     default: ""
     name: image-digest
     type: string
+  results:
+    - name: IMAGE_URL
+      value: "$(tasks.build-container.results.IMAGE_URL)"
+    - name: IMAGE_DIGEST
+      value: "$(tasks.build-container.results.IMAGE_DIGEST)"
+    - name: CHAINS-GIT_URL
+      value: "$(tasks.clone-repository.results.url)"
+    - name: CHAINS-GIT_COMMIT
+      value: "$(tasks.clone-repository.results.commit)"
```

Comment:
is mandatory to add these specific key-value pairs in this new "results" field? I remember sharing an error with you which complained about the absence of `CHAINS-GIT_URL` :thinking:

Reply:
Yes, this is the fix for that - the build pipeline is expected to have these results here.

Reply:
ack, thanks!